lib/game.rb

```diff
@@ -0,0 +1,183 @@
+# no structure yet, just drafting some logic
+# will split up into dif methods in the future
+
+class User
```

Comment:
I like that you are breaking out some classes.  As you continue to develop the game, I would think some more about the responsibilities of each class.  What is the user's role in the game?  What might be the role of some other class?  It makes me think of something like watching TV.  As a user of a television, you might be responsible for picking a channel, but you wouldn't be responsible for displaying the image.